src/v/cluster/cluster_discovery.cc

```diff
@@ -125,6 +125,17 @@ ss::future<bool> cluster_discovery::is_cluster_founder() {
     co_return *_is_cluster_founder;
 }
 
+cluster_discovery::node_ids_by_uuid& cluster_discovery::get_node_ids_by_uuid() {
```

Comment:
Thinking about this more, it's probably unnecessary to dodge the slow-founder bug since there's only one founder. But still it will probably be less surprising to make the behavior consistent.